### PR TITLE
add translate

### PR DESCRIPTION
**Pull request by kideny**
add translate

### CODE REVIEW

File: content/themes/.gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
-*
```
Comment (sergeyklay) [inline content/themes/.gitignore]: why

**Comment by kideny** [comment]
i install phanbook using git ，i  found the default theme is ignored。
site's index give me a error

**Comment by duythien** [comment]
Me too

**Comment by duythien** [comment]
I have no idea, I try clone again 

```
git clone https://github.com/phanbook/phanbook.git
```
After that I do check directory themes and see it

**Review by kideny — COMMENTED** [review]
I fixed

**Review by sergeyklay — APPROVED** [review]

**Comment by duythien** [comment]
Thanks

**Pull request merged**